NVIDIA-BaseOS-6/linux-nvidia-6.5
# UBUNTU: [Config] EFI: set CAPSULE_LOADER=y

BugLink: https://nvbugspro.nvidia.com/bug/4601764

[Description]
Nvidia provide a way to flash the UEFI via capsule loader.
CAPSULE_LOADER is also built-in in L4T kernel
so for the easy use, need to make CAPSULE_LOADER as built-in.

[Test Cases]
1. build kernel with nvidia-64k and nvidia flavours
~~~
* because there're mis-configured CONFIGs and annotation checking complains,  use do_skip_checks=true to ignore during compling.
time env CONCURRENCY_LEVEL=73 sh -c 'rm -f ../*.deb;fakeroot debian/rules clean && fakeroot debian/rules binary-headers binary-nvidia-64k do_linux_tools=false do_zfs=false do_mstflint_access=false do_nvidia-fs=false do_skip_checks=true' 2>&1 | tee build.log
~~~
3. boot kernel with the modification.
    * currently, use latest canonical kernel(6.5.0-1019-nvidia-64k) to verify
4. check if CONFIG_CAPSULE_LOADER is "y".
5. check if /dev/efi_capsule_loader exists.
~~~
# get nb4601764_verify.sh from [0]
$ sh nb4601764_verify.sh 
### Checking CONFIG_EFI_CAPSULE_LOADER in /boot/config-6.5.0-1019-nvidia-64k
CONFIG_EFI_CAPSULE_LOADER=y

### Checking /dev/efi_capsule_loader
/dev/efi_capsule_loader exists.
~~~

[0], verification scripts, http://nv/ecM

[Where things could go wrong]
Low,  minor affect with CONIFG_CAPSULE_LOADER=y
